iosHackthonApp/Modules/Features/MealList/Package.swift

```diff
@@ -0,0 +1,32 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "MealList",
+	platforms: [.iOS(.v14)],
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "MealList",
+            targets: ["MealList"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+		.package(path: "../Shared/Extensions"),
+		.package(path: "../Shared/Location"),
+		.package(path: "../Shared/Components"),
+		.package(name: "shared", path: "/Users/kanecheshire/Projects/benchHackathonNov20/shared/swiftpackage")
```

Comment:
will this be a problem that the path is specific to your machine?

Reply:
Oops! Yes, it's meant to be relative, I forgot to change. That explains why CI is broken!